### ISSUE TITLE
Paste deleted automatically even when no password have been entered

### ISSUE THREAD

**Issue by ghost** [PrivateBin/PrivateBin#628]
## Steps to reproduce

1. Create a paste with "Burn after reading" and password enabled
2. Go to the link with the just-created paste, but close the tab instead of entering the password
3. Go to the link again

### What happens
After closing the tab with the paste on the password screen, going to the link again caused this error to appear: 
> Could not get paste data: Paste does not exist, has expired or has been deleted.

### What should happen
After closing the tab with the paste on the password screen, going to the link again should prompt the user to insert the password, and then display the paste after a correct password have been entered.

## Additional information

Here is a YouTube video demonstrating the issue: https://youtu.be/FkWyI2t2MMo

## Basic information

**Server address**: bin.disroot.org

**Server OS:** N/A

**Webserver:** N/A

**Browser:** Firefox 76.0.1 (64-bit)

**PrivateBin version:** 1.3.4

I can reproduce this issue on <https://privatebin.net>: **Yes**

**Comment by rugk** [comment]
I have thought this issue is something we have tracked already, but I cannot find the dupe right now.

Anyway thanks for the video, even, and nice cursor… :wink: 

I guess we've closed this as dupes of https://github.com/PrivateBin/PrivateBin/issues/245, which is the actual solution for this bug.

However, given this nice bug report, I'd tend to leave this open, as it describes a problem and https://github.com/PrivateBin/PrivateBin/issues/245 is (actually) a feature request. Also people kind find it easier (so we don't get more dupes).
